internal/mock/cloudsql.go

```diff
@@ -72,20 +126,56 @@ func NewFakeCSQLInstance(project, region, name string, opts ...FakeCSQLInstanceO
 	}
 
 	f := FakeCSQLInstance{
-		project:   project,
-		region:    region,
-		name:      name,
-		ipAddrs:   map[string]string{"PUBLIC": "0.0.0.0"},
-		dbVersion: "POSTGRES_12", // default of no particular importance
-		Key:       key,
-		Cert:      cert,
+		project:     project,
+		region:      region,
+		name:        name,
+		ipAddrs:     map[string]string{"PUBLIC": "0.0.0.0"},
+		dbVersion:   "POSTGRES_12", // default of no particular importance
+		backendType: "SECOND_GEN",
+		sign:        SelfSign,
```

Comment:
nit: Field seems like it should be "signer" to be consistent with the option "WithSigner"

Reply:
Done.